docs/_docs/user-guide/golem.md

```diff
@@ -24,6 +24,126 @@ cargo build --release && \
     ../target/debug/golem ../../tests/golem_cli_test/tomes/hello_world.tome
 ```
 
+## Creating and testing tomes
+
+Golem is a great way to create and test tomes without deploying the entire Realm project.
+Golem operates in three different modes:
+
+- Interactive `-i`
+- With embedded tomes _no args_
+- On a specific tome `/path/to/tome/eldritch.main`
+
+In this guide we'll leverage the last one.
+
+### Tome structure
+
+Copy an existing tome and rename it to your desired name.
+
+```bash
+[~/realm]$ cd tavern/tomes/
```

Comment:
Maybe we should add a `golem init` command, to build the boiler plate for new tomes

Reply:
I don't love the idea of adding non operational stuff to golem and imix.
Instead we could add a template tome that users can copy out.